Mathlib/GroupTheory/DivisibleHull.lean

```diff
@@ -182,11 +182,13 @@ theorem qsmul_def (a : ℚ) (x : DivisibleHull M) :
 theorem zero_qsmul (x : DivisibleHull M) : (0 : ℚ) • x = 0 := by
   simp [qsmul_def]
 
+attribute [local instance] LocalizedModule.moduleOfIsLocalization in
```

Comment:
Would it be better to introduce the instance explicitly for `DivisibleHull` in this file?
```
instance : Module ℚ≥0 (DivisibleHull M) := LocalizedModule.moduleOfIsLocalization ..
```